manager/orchestrator/update/updater.go

```diff
@@ -155,34 +155,19 @@ func (u *Updater) Run(ctx context.Context, slots []orchestrator.Slot) {
 		u.startUpdate(ctx, service.ID)
 	}
 
-	delay := defaults.Service.Update.Delay
-	parallelism := int(defaults.Service.Update.Parallelism)
-	failureAction := defaults.Service.Update.FailureAction
-	allowedFailureFraction := defaults.Service.Update.MaxFailureRatio
-	monitoringPeriod, _ := gogotypes.DurationFromProto(defaults.Service.Update.Monitor)
-	order := defaults.Service.Update.Order
-
 	updateConfig := service.Spec.Update
 	if service.UpdateStatus != nil && service.UpdateStatus.State == api.UpdateStatus_ROLLBACK_STARTED {
 		updateConfig = service.Spec.Rollback
-	}
-
-	if updateConfig != nil {
-		failureAction = updateConfig.FailureAction
-		allowedFailureFraction = updateConfig.MaxFailureRatio
-		parallelism = int(updateConfig.Parallelism)
-		delay = updateConfig.Delay
-		order = updateConfig.Order
-
-		var err error
-		if updateConfig.Monitor != nil {
-			monitoringPeriod, err = gogotypes.DurationFromProto(updateConfig.Monitor)
-			if err != nil {
-				monitoringPeriod, _ = gogotypes.DurationFromProto(defaults.Service.Update.Monitor)
-			}
+		if updateConfig == nil {
+			updateConfig = defaults.Service.Rollback
 		}
+	} else if updateConfig == nil {
+		updateConfig = defaults.Service.Update
 	}
 
+	parallelism := int(defaults.Service.Update.Parallelism)
```

Comment:
In the previous code, it seemed like at this point the parallelism and monitoring period came from the current value of `updateConfig`, I guess because it could possible come from `service.Spec.Rollback`?  Should we set this to:

```
parallelism := int(updateConfig.Parallelism)
monitoringPeriod, err = gogotypes.DurationFromProto(updateConfig.Monitor)		
if err != nil {		
    monitoringPeriod, _ = gogotypes.DurationFromProto(defaults.Service.Update.Monitor)		
}
```

instead?